doc/src/Packages_details.rst

```diff
@@ -886,6 +887,22 @@ This package has :ref:`specific installation instructions <electrode>` on the
 
 ----------
 
+.. _PKG-EXTRA-COMMAND:
+
+EXTRA-COMMAND package
+---------------------
+
+**Contents:**
+
+Additional compute styles that are less commonly used.
```

Comment:
Did you mean specifically "compute styles" instead of "commands"?

Reply:
No. This is a copy-n-paste error.